src/utilities/replay/replay.cpp

```diff
@@ -156,9 +156,7 @@ void trim(std::string &s)
              s.end());
 }
 
-conduit::Node load_actions(const std::string &file_name, int mpi_comm_id){
-  conduit::Node actions;
-
+void load_actions(conduit::Node &actions, const std::string &file_name, int mpi_comm_id){
```

Comment:
Unwritten rule: prefer out params last :-)
```suggestion
void load_actions(const std::string &file_name, int mpi_comm_id, conduit::Node &actions)
{
```